var/spack/repos/builtin/packages/brainbuilder/package.py

```diff
@@ -13,7 +13,7 @@ class Brainbuilder(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/nse/brainbuilder"
 
     version('develop', branch='master')
-    version('0.11.2', tag='brainbuilder-v0.11.2', preferred=True)
+    version('0.11.3', tag='brainbuilder-v0.11.3', preferred=True)
```

Comment:
```suggestion
    version('0.11.3', tag='brainbuilder-v0.11.3')
```
The newest version is always preferred.

Reply:
I vaguely recall there was a catch with lexicographical sort order where `0.9` turned out to be "newer" than `0.10`; but maybe it was with `module load`.

Reply:
You're also referencing the version quite explicitly during deployment…

Reply:
Also true. ^^